float64 x
float64 y
float64 z
int8 cmd
int8 target_id
